playbook/app/pb_kits/playbook/utilities/_border_radius.scss

```diff
@@ -1,4 +1,14 @@
-@import "../tokens/exports/border_radius";
```

Comment:
Let's make sure these exports are accessible in some way. I believe these exports are used for js to access tokens. @jasperfurniss this is an area we need to improve in the file structure

Reply:
Yeah it's a good reminder -- I'll get Colin to add a story for making our scss tokens accessible in JS/TS. It's a good one to add, and will make us not have to do the `:export { }` syntax [in the client application](https://github.com/powerhome/nitro-web/blob/3515799f3179e1147e5f3d73b69ebc668e5b00b5/components/power_story/app/components/shared/Charts/typography.scss#L3). I think there's a way we can cleanly export that ability from playbook itself.